.github/workflows/master-e2e.yaml

```diff
@@ -39,6 +46,7 @@ on:
         type: string
       test_description:
         description: Short description of the test
+        default: unknown
```

Comment:
Sorry, just for my knowledge.
What is the purpose of explicitly setting `default: unknown`?

Reply:
Answered offline, changing it to **A description**